pymongo/mongo_client.py

```diff
@@ -645,6 +645,10 @@ def __init__(
         dbase = None
         opts = common._CaseInsensitiveDictionary()
         fqdn = None
+        # Raise an error when we have more than one URI
+        if len([i for i in host if "/" in i]) > 1:
+            raise ConfigurationError("MongoClient() only accepts 1 URI at a "
+                                     "time")
```

Comment:
I'd say: "host must not contain multiple MongoDB URIs"

Reply:
Done.